ocaml/xapi/xapi_vm_clone.ml

```diff
@@ -399,8 +400,8 @@ let clone ?(snapshot_info_record) disk_op ~__context ~vm ~new_name =
               List.iter (fun (vbd, newvdi, _) ->
                   let vbd = Xapi_vbd_helpers.copy ~__context ~vm:ref ~vdi:newvdi vbd in
                   (* CA-58405: when we make a clone/snapshot/checkpoint we consider the clone/snapshot/checkpoint VM
-                     					   to "own" all the clone/snapshot/checkpoint *disks* irrespective of the ownership of the original
-                     					   disks. We wish the clone/snapshot/checkpoint disks to be cleaned up with the VM. *)
+                     to "own" all the clone/snapshot/checkpoint *disks* irrespective of the ownership of the original
+                     disks. We wish the clone/snapshot/checkpoint disks to be cleaned up with the VM. *)
```

Comment:
The indentation changes should probably go into a separate commit, there's less noise that way when reviewing the main commit.